### PR TITLE
29 Increase allowable base width of editor combo-button

### PR DESCRIPTION
**Pull request by iperevozchikov**

### CODE REVIEW

File: src/styles/main.scss [hunk 1]
```diff
@@ -115,6 +115,13 @@ header.navbar .navbar-brand {
   display: block !important;
 }
 
+.cke_combo_button {
+  .cke_combo_text {
+    width: calc(100% - 29px) !important;
```
Comment (vanekbr) [inline src/styles/main.scss]: Usage of `!important` and absolute values is a bad practice.

The following code can be used to fix the issue:
```
#editor .cke_combo_button {
  .cke_combo_text {
    width: auto;
  }
}
```

Comment (iperevozchikov) [reply src/styles/main.scss]: But I would be limiting possible width by `max-width` to prevent very long buttons

Comment (iperevozchikov) [reply src/styles/main.scss]: :heavy_check_mark:

File: src/styles/main.scss [hunk 2]
```diff
@@ -115,6 +115,15 @@ header.navbar .navbar-brand {
   display: block !important;
 }
 
+.cke_combo {
```
Comment (vanekbr) [inline src/styles/main.scss]: Just realized it can be just in one line `.cke_combo .cke_combo_button .cke_combo_text {`

**Pull request merged**